### PR TITLE
feat: [SRM-11548]: Cypress AppDTemplate

### PR DESCRIPTION
**Pull request by deepesh-ui**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 102 Issues 
* Code issue 102 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10783)

File: cypress/integration/85-cv/HealthSourceTemplates/EmptyMonitoredService.spec.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+            lastUpdatedAt: 0
+          }
+        ]
+      })
+    })
+    cy.login('test', 'test')
+    cy.intercept('GET', monitoredServiceListCall, monitoredServiceListResponse)
+    cy.intercept(
+      'GET',
+      '/cv/api/monitored-service/count-of-services?routingId=accountId&accountId=accountId&orgIdentifier=default&projectIdentifier=project1',
```
Comment (harshilbits) [inline cypress/integration/85-cv/HealthSourceTemplates/EmptyMonitoredService.spec.ts:40]: please define the endpoints in constants

File: cypress/integration/85-cv/HealthSourceTemplates/EmptyMonitoredService.spec.ts [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+      cy.get('button[type="submit"]').click()
+    })
+    cy.contains('span', 'Template published successfully').should('be.visible')
+  })
+
+  it('Add new empty monitored service as fixed and environment as runtime', () => {
+    cy.intercept('GET', servicesCall, servicesResponse).as('ServiceCall')
+    cy.intercept('GET', environmentsCall, environmentResponse).as('EnvCall')
+    const applyTemplateCall =
+      '/template/api/templates/applyTemplates?routingId=accountId&accountIdentifier=accountId&orgIdentifier=default&projectIdentifier=project1&getDefaultFromOtherRepo=true'
```
Comment (harshilbits) [inline cypress/integration/85-cv/HealthSourceTemplates/EmptyMonitoredService.spec.ts:131]: same here.

Comment (deepesh-ui) [reply cypress/integration/85-cv/HealthSourceTemplates/EmptyMonitoredService.spec.ts]: done

File: cypress/integration/85-cv/HealthSourceTemplates/EmptyMonitoredService.spec.ts (hunk 1 above)
Comment (deepesh-ui) [reply cypress/integration/85-cv/HealthSourceTemplates/EmptyMonitoredService.spec.ts]: this was at multiple places , replaced all

**Review by harshilbits — APPROVED** [review]

**Review by praneshg239 — APPROVED** [review]

**Pull request merged**